troposphere/static/js/components/images/ImageTagsPage.react.js

```diff
@@ -112,20 +120,23 @@ export default React.createClass({
 
         return (
         <div className="container">
-            <div id="search-container">
```

Comment:
I'm wondering if missing the "search-container" ID here will cause trouble with Operation Sanity